docs-src/luna-streaming-core/modules/ROOT/pages/admin-console-VM.adoc

```diff
@@ -8,13 +8,13 @@ The Admin Console is a VueJS application that runs in a browser. It also include
 
 This document covers: +
 
-* xref:admin-console-VM.adoc#install[*Installation*] +
+* xref:admin-console-vm.adoc#install[*Installation*] +
```

Comment:
You should try to avoid using traditional xref syntax when targeting an anchor on the same page. Instead, you should use [internal xref](https://docs.antora.org/antora/latest/asciidoc/in-page-xref/) syntax. This helps writers more easily identify that the target is on the same page.

```suggestion
* <<install,*Installation*>> +
```